composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "crada/php-apidoc",
+    "name": "chxj1992/php-apidoc",
```

Comment:
? why would you change this?

Reply:
oh soooo  sorry,  I'm not mean to commit!

Reply:
I wanted to create a PR only to the first commit, I'm not very familiar with how to doing that !

Reply:
I changed the composer.json file because I wanted to import this awesome lib to my project and i can not wait. Is it possible for u to only merge the first commit?

Reply:
sorry, again

Reply:
don't worry! :)
you could simply commit again with the fix ;)